packages/atlas/src/components/Text/Text.styles.ts

```diff
@@ -49,6 +49,7 @@ const clampStyles = ({ clampAfterLine }: TextBaseProps) => css`
 
 const baseStyles = ({ color = 'colorTextStrong', clampAfterLine, margin, align }: TextBaseProps) => css`
   color: ${color === 'inherit' ? color : cVar(color)};
+  white-space: pre;
```

Comment:
I don't think we should apply this property to the Text's base styles. I already found some issues with text overflowing: 
![image](https://user-images.githubusercontent.com/51168865/205239635-8e22bef5-4adb-4d5b-bacb-9c8e71c004ae.png)
![image](https://user-images.githubusercontent.com/51168865/205239790-b82ed1df-6e57-4861-b938-e7230868edf8.png)